neofs-testlib/neofs_testlib/hosting/hosting.py

```diff
@@ -0,0 +1,108 @@
+import re
+from typing import Any
+
+from neofs_testlib.hosting.config import HostConfig, ServiceConfig
+from neofs_testlib.hosting.interfaces import Host
+from neofs_testlib.hosting.docker_host import DockerHost
+
+
+class Hosting:
+    """Hosting manages infrastructure  where neoFS runs (machines and neoFS services)."""
+
+    _hosts: list[Host]
+    _host_by_address: dict[str, Host]
+    _host_by_service_name: dict[str, Host]
+
+    @property
+    def hosts(self) -> list[Host]:
+        """Returns all hosts registered in the hosting.
+
+        Returns:
+            List of hosts.
+        """
+        return self._hosts
+
+    def configure(self, config: dict[str, Any]) -> None:
+        """Configures hosts from specified config.
+
+        All existing hosts will be removed from the hosting.
+
+        Args:
+            config: Dictionary with hosting configuration.
+        """
+        hosts = []
+        host_by_address = {}
+        host_by_service_name = {}
+
+        host_configs = [HostConfig(**host_config) for host_config in config["hosts"]]
+        for host_config in host_configs:
+            if host_config.plugin_name != 'docker':
+                raise ValueError("Only docker hosts are currently supported")
```

Comment:
You've changed these lines without a notice, please don't do this again. This MUST be a separate commit with appropriate annotation.

Reply:
fixed